tests/utils.py

```diff
@@ -7,6 +7,23 @@
 from typing import Optional
 
 
+@contextmanager
+def env(**kwargs) -> None:
+    """Context Manager to change env variable."""
+    tmp = {}
+    try:
+        for name, value in kwargs.items():
+            tmp[name] = os.environ.get(name)
+            os.environ[name] = value
+        yield
+    finally:
+        for name in kwargs.keys():
+            if tmp[name] is None:
+                os.environ.pop(name)
+            else:
+                os.environ[name] = tmp[name]
```

Comment:
Sorry just realised this can be simplified a little bit:

```suggestion
    original = dict(os.environ)
    try:
    	os.environ.update(kwargs)
        yield
    finally:
        os.environ.clear()
        os.environ.update(original)
```